package.json

```
@@ -8,7 +8,7 @@
     "build:dev:watch": "npm-run-all -p -l sass:watch webpack:dev:watch",
     "build:prd": "npm-run-all -p -l sass webpack:prd",
     "sass": "sass src/sass:src/dist/css",
-    "sass:watch": "npm run sass && sass src/sass:src/dist/css --watch --recursive",
+    "sass:watch": "sass --watch src/sass:src/dist/css src/Site/views/languageforge/theme/default/sass:src/dist/css src/angular-app:src/dist/css",
```

I am trying to figure out why the extra path mappings are needed.  I didn't see any comment in the PR about changes to sass compilation and my thinking is there shouldn't be a need for the second and third path mappings that you've included on the command line, since they are included as import statements in [`src/sass/languageforge.scss`](https://github.com/sillsdev/web-languageforge/blob/develop/src/sass/languageforge.scss#L1) 

First line in `languageforge.scss`:
```
@import "../Site/views/languageforge/theme/default/sass/languageforge_default.scss";
```
The other sass files in angular-app are explicitly imported one line at a time.

It seems fine to remove the `--recursive` since that doesn't actually seem necessary, as well as the redundant `npm run sass`.  Thanks for fixing that up.

```suggestion
    "sass:watch": "sass --watch src/sass:src/dist/css",
```

Shouldn't it work with the simplified suggestion above?  I haven't tested it.

---

sorry - please disregard this suggestion as it refers to #1159.

Unfortunately I reviewed this PR before #1159 and didn't realize that this PR included unrelated changes.  In the future please ensure PR change sets include only what is intended and not other PRs.  This requires a bit of git foo to arrange branches in certain ways.  There are some cases in which PRs must depend upon changes, however in this case these changes were unrelated (caddy vs sass) and could have been separated.

---

yep, you're right.  Taking the PR's in order is probably another good way to avoid the confusion.  Thanks.